pxr/usdImaging/usdImaging/lightAdapter.cpp

```diff
@@ -80,6 +80,42 @@ UsdImagingLightAdapter::_RemovePrim(SdfPath const& cachePath,
                                     UsdImagingIndexProxy* index)
 {
     index->RemoveSprim(HdPrimTypeTokens->light, cachePath);
+    UsdImaging_CollectionCache &collectionCache = _GetCollectionCache();
+    SdfPath lightLinkPath = cachePath.AppendProperty(UsdImagingTokens->collectionLightLink);
+    collectionCache.RemoveCollection(GetDelegate()->GetStage(), lightLinkPath);
+    SdfPath shadowLinkPath = cachePath.AppendProperty(UsdImagingTokens->collectionShadowLink);
+    collectionCache.RemoveCollection(GetDelegate()->GetStage(), shadowLinkPath);
```

Comment:
Might want to remove _collectionHashes[cachePath] here...

Reply:
Good point, but not needed anymore as I'm not storing the hashes any longer!